source/_integrations/bayesian.markdown

```diff
@@ -199,3 +207,66 @@ binary_sensor:
+
+This is an example sensor to detect if the bins have been left on the side of the road and need to be brought in closer to the house. It combines a theoretical presence sensor that gives a numeric signal strength and an API sensor from local government that can have 3 possible states: `due` when collection is due in the next 24 hours, `collected` when collection has happened in the last 24 hours and `not_due` at other times.
+
+```yaml
+# Example configuration.yaml entry
+binary_sensor:
+  name: "Bins need bringing in"
+  platform: "bayesian"
+  prior: 0.14 # bins are left out for usually about one day a week
+  probability_threshold: 0.5
+  observations:
+    - platform: "numeric_state"
+      entity_id: "sensor.signal_strength"
+      prob_given_true: 0.01 # if the bins are out and need bringing in there is only a 1% we will get a strong signal of above 10
+      prob_given_false: 0.3 # if the bins are not out, we still tend not to get a signal this strong
+      above: 10
+    - platform: "numeric_state"
+      entity_id: "sensor.signal_strength"
+      prob_given_true: 0.02
+      prob_given_false: 0.5 #if the bins are not out we often get a signal this strong
+      above: 5
+      below: 10
+    - platform: "numeric_state"
+      entity_id: "sensor.signal_strength"
+      prob_given_true: 0.07
+      prob_given_false: 0.1
+      above: 0
+      below: 5
+    - platform: "numeric_state"
+      entity_id: "sensor.signal_strength"
+      prob_given_true: 0.3
+      prob_given_false: 0.07
+      above: -10
+      below: 0
+    - platform: "numeric_state"
+      entity_id: "sensor.signal_strength"
+      prob_given_true: 0.6 #if the bins are out we often get a signal this weak or even weaker
+      prob_given_false: 0.03
+      below: -10
+    # then lets say we want to combine this with an imaginary sensor.bin_collection which reads for a local government API which can have 1 of three values (collected, due, not due)
+    - platform: "state"
+      entity_id: "sensor.bin_collection"
+      prob_given_true: 0.8 # If the bins need bringing in, usually its because they've just been collected
+      prob_given_false: 0.05 # 
+      to_state: "collected"
+    - platform: "state"
+      entity_id: "sensor.bin_collection"
+      prob_given_true: 0.05 # If the bins need bringing in then the sensor.bin_collection shouldn't be 'due'
+      prob_given_false: 0.11 # The sensor will be 'due' for about 1 day a week whilst for the 24 hours before collection
+      to_state: "due"
+    - platform: "state"
+      entity_id: "sensor.bin_collection"
+      prob_given_true: 0.15 #All the prob_given_true should add to 1
+      prob_given_false: 0.84 # All the prob_given_false should add to 1
+      to_state: "not due"
+```
+To achieve a similar thing for multiple `template` observations targeting a single entity, simply write your templates to return `True` or `None` rather than `True` or `False`.
```

Comment:
```suggestion
To achieve a similar result for multiple `template` observations targeting a single entity, write your templates to return `True` or `None` rather than `True` or `False`.
```